Api/Client.php

```diff
@@ -494,28 +401,23 @@ private function parseSoapFault(\Exception $e)
             }
         }
         $errors = is_array($error) ? $error : ['error' => $error];
+//        var_dump($errors);
```

Comment:
Also code style above is off, inconsistent usage of { and newlines, please use PSR formatting